.pre-commit-config.yaml

```diff
@@ -19,13 +19,12 @@ repos:
   hooks:
   - id: isort
     args:
+    # Keep in sync with pyproject.toml
+    - --force-single-line-imports
     - --honor-noqa
-    exclude: |
-      (?x)
-      ^
-        .*
-      $
-    stages: ["manual"]
+    - --lines-between-types=1
+    - --profile=black
+    - --reverse-relative
```

Comment:
What is this?

Reply:
I think you mean the reverse-relative? "Reverse order of relative imports."  

the default does:
...grandparent
..parent
.sibling

this reverses that

.sibling
..parent
...grandparent

it was does to minimize the changes in the repo, I didn't capture the exact count.  

My preference here would be to revist the config after it is in place, minimizing the current changes now.

I'm happy to log an issue here, so it is not lost or forgotten

Reply:
Just make sure to explain the motivation. It probably makes sense to have it so that there are fewer potential import loops.

Reply:
@webknjaz This was logged and linked so it is not forgotten in the future:

https://github.com/ansible/ansible-navigator/issues/780

I see this as resolved, can you close it or follow-up with remaining concerns/requested changes?

Reply:
resolving conversation, please reopen if the concerns and question are unanswered, ty for the feedback